ftl/common/tar_to_dockerimage.py

```diff
@@ -123,16 +123,21 @@ def config_file(self):
                 self._overrides.pop('Entrypoint', None)
             env = {}
             if 'Env' in self._overrides:
-                entrypoint = self._overrides['Env']
+                env = self._overrides['Env']
```

Comment:
I think you can clean these up with env = self._overrides.pop('Env', None)

Reply:
done